src/Vector4_TEST.cc

```diff
@@ -57,6 +57,10 @@ TEST(Vector4dTest, Vector4d)
   v.Normalize();
   EXPECT_EQ(v, math::Vector4d(0.182574, 0.365148, 0.547723, 0.730297));
 
+  // ::Normalized
+  v.Set(1, 2, 3, 4);
+  EXPECT_EQ(v.Normalized(), math::Vector4d(0.182574, 0.365148, 0.547723, 0.730297));
```

Comment:
The code checker is complaining that this line is too long:

`./src/Vector4_TEST.cc:62:  Lines should be <= 80 characters long  [whitespace/line_length] [2]`

You can see the results here: https://github.com/ignitionrobotics/ign-math/pull/140/checks?check_run_id=932511831

You can run the code checker locally with:

```
cd <path to ign-math>
bash tools/code_check.sh
```

Reply:
Forget to use code checker. To resolve I just break the line putting the second argument under the first like in Style Guide.